### ISSUE TITLE
checkpoint

### ISSUE THREAD

**Issue by xiaoweimingood** [qhfan/RMT#10]
Nice work! Could you release the Imagenet checkpoint?

**Comment by qhfan** [comment]
Thanks for your attention to our work. The ckpt will be released after we finish organizing it

**Comment by HJY618** [comment]
Thank you for your nice work! Could you release the Imagenet pre_trained weight of RMT-S, RMT-B and RMT-L? Thank you again!

**Comment by Snowwwwwwwwwww** [comment]
You may address the author's other projects, but none of these projects have provided complete details, such as pre-trained weights. Despite numerous GitHub issues, the authors have chosen to disregard them😅. This raises doubts about whether the authors actually conducted any experiments for their published works🤨.

Authors should prioritize taking responsibility for their open-source projects, particularly in the context of papers published in prestigious conferences. @qhfan

**Comment by qhfan** [comment]
Thank you for your attention to our work. We have many projects underway, which has delayed the organization of the code and weights. We sincerely apologize for any inconvenience this has caused you. Additionally, I do not believe that it is very polite to casually question someone else's work, especially given that we have made the training logs publicly available. @Snowwwwwwwwwww

**Comment by qhfan** [comment]
Thank you for your attention to our work. We have many projects underway, which has delayed the organization of the code and weights. We sincerely apologize for any inconvenience this may have caused you. The weights are now open-sourced, and we have uploaded the original code as well. Additionally, I do not believe that it is very polite to casually question someone else's work, especially when we have made the training logs publicly available. @Snowwwwwwwwwww
